invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/components/CommunityHeader/CommunityHeader.js

```diff
@@ -54,6 +56,10 @@ class CommunityHeaderComponent extends Component {
                   {community.metadata.title}
                 </div>
               </>
+            ) : permissions?.can_publish_restricted ? (
+              <Message visible warning className="mt-15" style={{ marginLeft: "15px" }}>
+                {i18next.t("Community is required to submit your record.")}
+              </Message>
             ) : (
```

Comment:
The rationale behind using the Message component is to draw the user's attention, encouraging them to select a community. However, I am open to alternative suggestions or approaches to achieve this objective.